### DISCUSSION TITLE
Testing Budgie Desktop with deepin-wm compositor

### DISCUSSION THREAD

**Discussion by bernku** [BuddiesOfBudgie/budgie-desktop#336] [Ideas]
It is possible to implement this wm to budgie desktop? is a lot smoother then budgie-wm im just testing and it works great, what do you think guys :)
heres is the link of the video i made testing the wm
https://youtu.be/j_-10_qsYgs

**Comment by JoshStrobl** [comment]
Our plan is to only support budgie-wm at this time.
